handbook/marketing/content-style-guide.md

```diff
@@ -108,12 +108,9 @@ As we use sentence case, only the first word is capitalized. But, if a word woul
   - "Zach is the co-creator of osquery"
   - "Does it work with MySQL?"
 
-#### Why sentence case?
-The reason for sentence case at Fleet is that everyone capitalizes differently in English, and capitalization conventions have not been taught very consistently in schools.  Sentence case simplifies capitalization rules so that contributors can deliver more natural, even-looking content with a voice that feels similar no matter where you're reading it.
+> Struggling with this? It takes some adjustment, and you need repetitions of seeing things written this way and correcting yourself. Many contributors have given the feedback that this opinionated solution is a huge relief once you build the habit of using sentence case capitalization, since it frees up mental capacity in every copywriting situation. You don't have to think as hard, nor choose between flouting and laboriously adhering to the (likely somewhat complex and out of date) styleguide.
 
-> Struggling with this?  It takes some adjustment, and you need repetitions of seeing things written this way and correcting yourself.  Many contributors have given the feedback that this opinionated solution is a huge relief once you build the habit of using sentence case capitalization, since it frees up mental capacity in every copywriting situation.  You don't have to think as hard, nor choose between flouting and laboriously adhering to the (likely somewhat complex and out of date) styleguide.
-
-> TODO: extrapolate the bulk of this whole sentence-case section to "Why this way", since it applies for every department at Fleet
+You can read about why we use sentence case in ["📖Company/Why this way?"](./../company/why-this-way.md#why-does-fleet-use-sentence-case).
```

Comment:
We can use the fleetdm.com equivalent to avoid accidents incase this is moved later